moped-editor/src/queries/components.js

```diff
@@ -0,0 +1,18 @@
+import { gql } from "@apollo/client";
+
+export const GET_COMPONENTS_FORM_OPTIONS = gql`
+  query GetComponentsFormOptions {
+    moped_components(
+      order_by: [{ component_name: asc }, { component_subtype: asc }]
```

Comment:
Just saw this and I pushed code to filter those out. Thanks for catching that and I guess we need to make another DB update to get aligned with the rest of the soft delete comments.

I was also happy to see this query come together - I hadn't ordered by multiple fields in the past so that was a TIL for me.

Reply:
Agreed. Added to the epic: https://github.com/cityofaustin/atd-data-tech/issues/10516